src/charm.py

```diff
@@ -168,22 +177,23 @@ def are_writes_running(self) -> bool:
             return False
 
     def _on_start(self, event: StartEvent) -> None:
-        """Only sets an Active status."""
-        self.unit.status = ActiveStatus()
+        """Sets initial Waiting status and checks if writes should be restarted."""
+        self.unit.status = BlockedStatus("Waiting for integration")
```

Comment:
Thank you for reacting on Marcelo comment about Waiting=>Blocked switch, but the Juju status message is still confusingly saying `waiting`, which may confuse end-users. 

Let's hint them to act: 
`Integrate with the database provider` (e.g. data-integrator)
or
`No database relation available` (e.g. sysbench charm)